bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/resolver/ActorProcessDependencyDeployer.java

```diff
@@ -81,13 +76,11 @@ public boolean deploy(final TenantServiceAccessor tenantAccessor, final Business
         }
         try {
             actorMappingService.addActors(sActors);
-            final byte[] actorMappingXML = businessArchive.getResource(ActorMappingContribution.ACTOR_MAPPING_FILE);
-            if (actorMappingXML != null) {
-                final String actorMapping = new String(actorMappingXML);
-                final ImportActorMapping importActorMapping = new ImportActorMapping(actorMappingService, identityService, parser, processDefinition.getId(),
-                        actorMapping);
+            ActorMapping actorMapping = businessArchive.getActorMapping();
+            if (actorMapping != null) {
+                final ImportActorMapping importActorMapping = new ImportActorMapping(actorMappingService,identityService);
                 try {
-                    importActorMapping.execute();
+                    importActorMapping.execute(actorMapping,processDefinition.getId());
                 } catch (final SBonitaException e) {
                     // ignore
```

Comment:
I know that you did not do this but we can ignore an exception on import without at least logging something